PublicMapping/districtbuilder
# Allow a user to create a project from an organization's template from the Create Project screen

## Overview

- Refactors `OrganizationTemplates` and `TemplateCard` into a new component outside of `OrganizationScreen`
- Renders `OrganizationTemplates` component in `CreateProjectScreen` to allow users to create a project from a template

### Checklist


### Demo

### Notes

- I still haven't tested this with multiple templates / multiple organizations, but I see no reason why it shouldn't work, conceptually. May need to do a bit of styling fix-up, but I think it's in the right direction currently.
- There are gonna be some gnarly merge conflicts with #633 here on `OrganizationScreen`. Am wondering which way it will be easier to resolve these - since the refactor of `OrganizationTemplates` is in this branch, it might make sense to merge this one in first?


## Testing Instructions

- With a user who has joined an organization that has at least one template - click 'New Map' from the home screen
- Expect: list of organizations you are a part of to be displayed
- Select the radio button for one of your organizations
- Expect: organization project templates displayed
- Click on a template to create a new project
- Expect: new project created from template

Closes #598
